tox.ini

```
@@ -0,0 +1,13 @@
+[flake8]
+application-package-names = enable,kiva,pyface,traits,traitsui
```

do we want to add the `application-import-names` and put `enable` and `kiva` under there instead? -> This is bound to raise a lot of new flake8 errors though.

```suggestion
application-package-names = pyface,traits,traitsui
application-import-names = enable,kiva
```

---

Good catch